binaries/cuprated/src/blockchain/interface.rs

```diff
@@ -21,18 +21,15 @@ use cuprate_types::{
 };
 
 use crate::{
-    blockchain::manager::BlockchainManagerCommand, constants::PANIC_CRITICAL_SERVICE_ERROR,
+    blockchain::manager::{BlockchainManagerCommand, IncomingBlockOk},
+    constants::PANIC_CRITICAL_SERVICE_ERROR,
 };
 
-/// The channel used to send [`BlockchainManagerCommand`]s to the blockchain manger.
-pub static COMMAND_TX: OnceLock<mpsc::Sender<BlockchainManagerCommand>> = OnceLock::new();
-
-/// A [`HashSet`] of block hashes that the blockchain manager is currently handling.
+/// The channel used to send [`BlockchainManagerCommand`]s to the blockchain manager.
 ///
-/// This is used over something like a dashmap as we expect a lot of collisions in a short amount of
-/// time for new blocks so we would lose the benefit of sharded locks. A dashmap is made up of `RwLocks`
-/// which are also more expensive than `Mutex`s.
-pub static BLOCKS_BEING_HANDLED: OnceLock<Mutex<HashSet<[u8; 32]>>> = OnceLock::new();
+/// This channel is initialized in [`init_blockchain_manager`](super::manager::init_blockchain_manager), the functions
+/// in this file document what happens if this is not initialized when they are called.
+pub(super) static COMMAND_TX: OnceLock<mpsc::Sender<BlockchainManagerCommand>> = OnceLock::new();
```

Comment:
This `(super)` is fine but I would have preferred if it were moved to `manager.rs` and kept private - it is still writeable from `interface.rs` since it's defined there, even though it only needs a `&mpsc::Sender`.

Doesn't matter too much I guess but then again things being scoped is nice.